src/make_app.ts

```diff
@@ -40,8 +40,14 @@ export function makeApp(options: AppOptions): PolyserveApplication {
   options = options || {};
   let root = options.root;
   let componentDir = options.componentDir || 'bower_components';
-  let packageName = options.packageName || bowerConfig(root).name
-      || path.basename(process.cwd());
+  let packageName = options.packageName;
+  if (!packageName){
+    packageName = bowerConfig(root).name;
+  }
+  if (!packageName) {
+    console.log('bower.json not found, falling back to current path as package name');
```

Comment:
Can we reword this to sound less error-like? We want this to work for application builds as well as elements.

How about:

```js
packageName = path.basename(process.cwd());
console.log(`no bower.json detected, using name "${packageName}"`);
```